stylesheet/components/card.css

```diff
@@ -0,0 +1,28 @@
+/* card box shadow */
+
+.card-shadow-xs {
+    box-shadow: 0 0 4px -2px;
+}
+
+
+/* card container dimensions */
+
+.card-dim {
+    width: max-content;
+    height: max-content;
+}
+
+
+/* card text widths */
+
+.card-txtw-xs {
+    width: 20ch;
```

Comment:
any specific reason for using ch here?

Reply:
yes, I wanted to keep the width of the card dependent on image or the text it has. Unlike before where I had hardcoded it.